geomesa-convert/geomesa-convert-common/src/main/scala/org/locationtech/geomesa/convert/Transformers.scala

```diff
@@ -226,7 +226,7 @@ object Transformers extends EnhancedTokenParsers with LazyLogging {
       if (idx == -1) {
         idx = ctx.indexOf(n)
       }
-      ctx.get(idx)
+      if (idx < 0) null else ctx.get(idx)
```

Comment:
would this work to avoid the check altogether?:
```scala
var doEval: EvaluationContext => Any = { ctx =>
      val idx = ctx.indexOf(n)
      doEval =
        if(idx < 0)  _  => null
        else         ec => ec.get(idx)
      doEval(ctx)
    }
    override def eval(args: Array[Any])(implicit ctx: EvaluationContext): Any = doEval(ctx)
```